index.bs

```diff
@@ -4898,96 +4898,117 @@ No [=extended attributes=] are applicable to dictionaries.
 
 <h3 id="idl-exceptions">Exceptions</h3>
 
-An <dfn id="dfn-exception" export>exception</dfn> is a type of object that
-represents an error and which can be thrown or treated as a first
-class value by implementations.  Web IDL does not allow exceptions
-to be defined, but instead has a number of pre-defined exceptions
-that specifications can reference and throw in their definition of
-operations, attributes, and so on.  Exceptions have an
-<dfn id="dfn-exception-error-name" for="exception" export>error name</dfn>,
-a {{DOMString}},
-which is the type of error the exception represents, and a
-<dfn id="dfn-exception-message" for="exception" export>message</dfn>, which is an optional,
-user agent-defined value that provides human readable details of the error.
-
-There are two kinds of exceptions available to be thrown from specifications.
-The first is a <dfn id="dfn-simple-exception" export>simple exception</dfn>, which
-is identified by one of the following types:
+An <dfn id="dfn-exception" export>exception</dfn> is a type of object that represents an error and
+which can be thrown or treated as a first class value by implementations. Web IDL has a number of
+pre-defined exceptions that specifications can reference and throw in their definition of
+operations, attributes, and so on. Custom exception types can also be defined, as [=interfaces=]
+that [=interface/inherit=] from {{DOMException}}.
+
+A <dfn id="dfn-simple-exception" export>simple exception</dfn> is identified by one of the
+following types:
 
 *   <dfn exception>EvalError</dfn>
 *   <dfn exception>RangeError</dfn>
 *   <dfn exception>ReferenceError</dfn>
 *   <dfn exception>TypeError</dfn>
 *   <dfn exception>URIError</dfn>
 
-These correspond to all of the ECMAScript [=ECMAScript/error objects=]
-(apart from <l spec=ecmascript>{{SyntaxError}}</l> and {{Error}},
-which are deliberately omitted as they are reserved for use
-by the ECMAScript parser and by authors, respectively).
-The meaning of each [=simple exception=] matches
-its corresponding error object in the
-ECMAScript specification.
+These correspond to all of the ECMAScript [=ECMAScript/error objects=] (apart from
+<l spec=ecmascript>{{SyntaxError}}</l> and <l spec=ecmascript>{{Error}}</l>, which are deliberately
```

Comment:
pre-existing, but: AggregateError, too.

(Given there’s at least [one more in the pipeline](https://arai-a.github.io/ecma262-compare/snapshot.html?pr=3000#sec-suppressederror-constructor), maybe keeping this in sync is too awkward and the gist could be conveyed without enumerating the excluded constructors...?)

Reply:
Please either file a follow-up or address.

Reply:
@annevk distinct issue here: https://github.com/whatwg/webidl/issues/1314